src/lane_detection/UNet-LaneDetection/lane_fit.py

```diff
@@ -5,7 +5,7 @@
 import matplotlib.pyplot as plt
 import time
 
-img_pth = '/Users/jasonyuan/Desktop/Test9.png'
+img_pth = '/Users/jasonyuan/Desktop/Test2.png'
```

Comment:
Please upload your test images so that we can run it out of the box!

If you know the path relative to the script, but not the cwd, then [this may help](https://stackoverflow.com/questions/918154/relative-paths-in-python)